examples/auth0/pages/api/protected-api.js

```diff
@@ -0,0 +1,25 @@
+import { withApiAuthRequired, getSession } from '@auth0/nextjs-auth0'
+
+async function handle(req, res) {
+  const { user } = getSession(req, res)
+
+  if (!user) {
```

Comment:
You don't need this check with the `withApiAuthRequired`

Reply:
Definitely, thanks mate